File: Timesheet.API/Models/Projet.cs
using System.Collections.Generic;

namespace Timesheet.API.Models
{
    public class Projet
    {
        public int ID {get; set;}
        public string Name {get; set;}
        public string Type {get; set;}
        public float Price {get; set;}
        public int ClientId {get; set;}

        public ICollection<Day> Days {get; set;}
    }
}

File: Timesheet-SPA/src/app/projets/models/projet.ts
import { Day } from 'src/app/day/model/day';

export class Projet {

  days: Day[];

  constructor(
    public name: string,
    public type: string,
    public price: number,
    public clientId: number,
    
  ) {
    this.id = 0;
    this.days = Array<Day>();
  }

  id: number;
}
